<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>865</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QGroupBox" name="groupBoxSort">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="title">
              <string>排序</string>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>排序字段</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1" colspan="2">
               <widget class="QComboBox" name="comboFields">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>65</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="1" column="1">
               <widget class="QRadioButton" name="radioBtnAscend">
                <property name="text">
                 <string>升序</string>
                </property>
               </widget>
              </item>
              <item row="1" column="2">
               <widget class="QRadioButton" name="radioBtnDescend">
                <property name="text">
                 <string>降序</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBoxFilter">
             <property name="font">
              <font>
               <family>微软雅黑</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="title">
              <string>数据过滤</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="0" column="0">
               <widget class="QRadioButton" name="radioBtnMan">
                <property name="text">
                 <string>男</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QRadioButton" name="radioBtnWoman">
                <property name="text">
                 <string>女</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QRadioButton" name="radioBtnBoth">
                <property name="text">
                 <string>全显示</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>41</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="tableView"/>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="font">
        <font>
         <family>微软雅黑</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>工 号</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="dbSpinEmpNo"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>姓 名</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="dbEditName"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>性 别</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="dbComboSex">
            <item>
             <property name="text">
              <string>男</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>女</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_5">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>身高</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QDoubleSpinBox" name="dbSpinHeight"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_6">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>出生日期</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QDateEdit" name="dbEditBirth">
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_7">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>手机号</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="dbEditMobile"/>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_8">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>出生省份</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QComboBox" name="dbComboProvince">
            <property name="editable">
             <bool>true</bool>
            </property>
            <item>
             <property name="text">
              <string>北京</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>上海</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>天津</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>重庆</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>安徽</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>河北</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>河南</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>湖北</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>湖南</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>山西</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>山东</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>陕西</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>江苏</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_9">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>城 市</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLineEdit" name="dbEditCity"/>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_10">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>部 门</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QComboBox" name="dbComboDep">
            <property name="editable">
             <bool>true</bool>
            </property>
            <item>
             <property name="text">
              <string>销售部</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>技术部</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>生产部</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>行政部</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="label_11">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>学 历</string>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="QComboBox" name="dbComboEdu">
            <property name="editable">
             <bool>true</bool>
            </property>
            <item>
             <property name="text">
              <string>博士研究生</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>硕士研究生</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>大学本科</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>大学专科</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>高中</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>初中</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>小学</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="10" column="0">
           <widget class="QLabel" name="label_12">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>工资</string>
            </property>
           </widget>
          </item>
          <item row="10" column="1">
           <widget class="QSpinBox" name="dbSpinSalary"/>
          </item>
          <item row="11" column="0">
           <widget class="QLabel" name="label_13">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>备注</string>
            </property>
           </widget>
          </item>
          <item row="11" column="1">
           <widget class="QPlainTextEdit" name="dbEditMemo"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>照片</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="dbLabPhoto">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>350</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>dbLabPhoto</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>328</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>865</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actOpenDB"/>
   <addaction name="actRecAppend"/>
   <addaction name="actRecInsert"/>
   <addaction name="actRecDelete"/>
   <addaction name="separator"/>
   <addaction name="actPhoto"/>
   <addaction name="actPhotoClear"/>
   <addaction name="actScan"/>
   <addaction name="separator"/>
   <addaction name="actSubmit"/>
   <addaction name="actRevert"/>
   <addaction name="separator"/>
   <addaction name="actQuit"/>
  </widget>
  <action name="actOpenDB">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/open3.bmp</normaloff>:/images/images/open3.bmp</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="toolTip">
    <string>打开数据库</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/exit.bmp</normaloff>:/images/images/exit.bmp</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
  <action name="actRecAppend">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/316.bmp</normaloff>:/images/images/316.bmp</iconset>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
   <property name="toolTip">
    <string>添加记录</string>
   </property>
  </action>
  <action name="actRecInsert">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/306.bmp</normaloff>:/images/images/306.bmp</iconset>
   </property>
   <property name="text">
    <string>插入</string>
   </property>
   <property name="toolTip">
    <string>插入记录</string>
   </property>
  </action>
  <action name="actSubmit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/save1.bmp</normaloff>:/images/images/save1.bmp</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="toolTip">
    <string>保存修改</string>
   </property>
  </action>
  <action name="actRevert">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/ubdo.bmp</normaloff>:/images/images/ubdo.bmp</iconset>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
   <property name="toolTip">
    <string>取消西裤该</string>
   </property>
  </action>
  <action name="actRecDelete">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/delete1.bmp</normaloff>:/images/images/delete1.bmp</iconset>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
   <property name="toolTip">
    <string>删除记录</string>
   </property>
  </action>
  <action name="actPhoto">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/00.JPG</normaloff>:/images/images/00.JPG</iconset>
   </property>
   <property name="text">
    <string>设置照片</string>
   </property>
   <property name="toolTip">
    <string>设置照片</string>
   </property>
  </action>
  <action name="actPhotoClear">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/103.bmp</normaloff>:/images/images/103.bmp</iconset>
   </property>
   <property name="text">
    <string>清除照片</string>
   </property>
   <property name="toolTip">
    <string>清除照片</string>
   </property>
  </action>
  <action name="actScan">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/up.bmp</normaloff>:/images/images/up.bmp</iconset>
   </property>
   <property name="text">
    <string>涨工资</string>
   </property>
   <property name="toolTip">
    <string>涨工资</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
